### ISSUE TITLE
Looking for new maintainer

### ISSUE THREAD

**Issue by andig** [andig/videodb#135]
I've not used videodb for a long time and can no longer dedicate sufficient time to maintain it. To improve it would be nice to find a new maintainer that I could give commit rights to.

Please let me know if you'd be interested.

/cc @kec42 @copperhead57

**Comment by johanneskonst** [comment]
Same...
I just restarted using this, would like to make it PHP 8+ compatible, perhaps add thetvdb api engine, even found a few security bugs... what would be the best way forward?

**Comment by andig** [comment]
@kec42 has merge access. Best way forward is probably contributing actual PRs.

**Comment by kec42** [comment]
I don't have merge access. I have just seen the invitation and it has expired.

**Comment by johanneskonst** [comment]
Yeah, I can contribute PRs but I think it would be wise to plan and discuss before making and submitting those...
By the looks of it there are three forks of interest, @kec42 's, one by @kec2 and one by @copperhead57
Should those then be part of any roadmap discussion or are they just forks for own use?

**Comment by kec2** [comment]
Hey

Kec2 isn't private as such but not so easy to merge back into videodb. 
Kec42 is only for making bug fixes for videodb.

On 16 July 2022 23.10.54 CEST, Johannes Konst ***@***.***> wrote:
>Yeah, I can contribute PRs but I think it would be wise to plan and discuss before making and submitting those...
>By the looks of it there are three forks of interest, @kec42 's, one by @kec2 and one by @copperhead57
>Should those then be part of any roadmap discussion or are they just forks for own use?
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/andig/videodb/issues/135#issuecomment-1186293845
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
Hilsen
Klaus Christiansen

**Comment by andig** [comment]
Is anyone looking into merging the piled prs? Missing rights to do so?

**Comment by johanneskonst** [comment]
I think @kec42 missed his accept-invitation-window so I think the answer to that question would be a "no, yes" ;-)
I would be happy to help as well in maintaining (and extending) but don't want to step on the proverbial toes, so to speak...

**Comment by andig** [comment]
@kec2 you have a new invite- I'll follow here in case you agree to spread on more shoulders.

**Comment by kec42** [comment]
I can help I just don't have much time.
More maintainers would be great.

**Comment by johanneskonst** [comment]
@andig @kec42 what would be a good way forward here?
i've submitted a few PR's and would like to build upon those but i'm a bit in limbo when they don't get accepted.
i'd also like to see kec's work with php8 compat and better testing added if possible.
and my own little pet peeve ('better' tv-show integration) which is what i would like to work on.

what would be a good way forward here? looking forward to your ideas...

**Comment by andig** [comment]
Seems we're not making progress. If someone want to volunteer, please get in touch here. Otherwise I'd like to put the repo into archive mode by end of the year.

**Comment by johanneskonst** [comment]
@andig I'm volunteering.
I would love to maintain this, I just don't want to step on the proverbial toes so to speak.
So unless @kec42 objects please make him and myself maintainers, we'll go from there...

Thanks!

**Comment by andig** [comment]
Sent invites to @johanneskonst @copperhead57.

**Comment by andig** [comment]
Invites confirmed. Welcome as new maintainers!
